G/GAP_pkg/bundled/gac-darwin

```diff
@@ -66,12 +66,12 @@ gap_compiler="${abs_top_builddir}/gap"
 . "${abs_top_builddir}/sysinfo.gap"
 
 # These three should be filled in by the standard autoconf procedures
-c_compiler="gcc -fno-common -DPIC"
-cxx_compiler="g++ -fno-common -DPIC"
+c_compiler="clang -fno-common -DPIC"
+cxx_compiler="clang++ -fno-common -DPIC"
 c_linker="echo static linking not supported ; exit 1 ;"
 
 # These will need special care
-c_dyn_linker="gcc -bundle -lgap"
+c_dyn_linker="clang -bundle -lgap"
```

Comment:
I'd suggest using the more generic `cc` and `c++`, which are good for all platforms

Reply:
Good point, done